events/events.go

```diff
@@ -58,20 +58,22 @@ func routeEvent(msg *events.Envelope) {
 	}
 }
 
-func SetupEventRouting(wantedEvents string) {
+func SetupEventRouting(wantedEvents string) error {
 	selectedEvents = make(map[string]bool)
-	for _, event := range strings.Split(wantedEvents, ",") {
-		if isAuthorizedEvent(event) {
-			selectedEvents[event] = true
-			log.LogStd(fmt.Sprintf("Event Type [%s] is included in the fireshose!", event), false)
-		} else {
-			log.LogError(fmt.Sprintf("Rejected Event Name [%s] - See wanted/selected events arg. Check Your Command Line Arguments!", event), "")
-		}
-	}
-	// If any event is not authorize we fallback to the default one
-	if len(selectedEvents) < 1 {
```

Comment:
Ah, thanks.

With this PR if you enter a bad event name, in any place in --events firehose-to-syslog will exit -1 and print a error message "Rejected Event Name [%s] ... ".

I think this is a better approach than falling back to a "default" event, so there is no ambiguity as to why you get an event you did not ask for and why you dont get what you actually wanted.

Also, just noticed, the error message should be better, currently